packages/suite/src/actions/wallet/__tests__/discoveryActions.test.ts

```diff
@@ -631,18 +633,21 @@ describe('Discovery Actions', () => {
             useEmptyPassphrase: false, // mandatory
         });
         const store = initStore(state);
-        const fn = discoveryActions.getDiscoveryAuthConfirmationStatus;
+
+        // Necessary workaround, as redux-mock-store, as we use it, doesn't have immutable state.
+        // TODO: We really should mock redux in unit tests differently.
+        const fn = (state: any) => selectDiscoveryAuthConfirmation({ ...state });
```

Comment:
Shouldn't we change the name of this `it` test to `selectDiscoveryAuthConfirmation`?
It is still `getDiscoveryAuthConfirmationStatus` referencing the previous one that does not exist anymore.

Reply:
Of course we should! Thank you.
And sorry that I've already squashed the fixup.